site/content/docs/_index.md

```diff
@@ -8,11 +8,8 @@ menu:
     weight: 20
 ---
 
-Kopia is a fast and secure open-source backup/restore tool that manages filesystem snapshots in a remote storage.
+Kopia is a fast and secure open-source backup/restore tool that allows you to create [encrypted](https://kopia.io/docs/features#end-to-end-zero-knowledge-encryption) snapshots of your data and save the snapshots to [remote or cloud storage](https://kopia.io/docs/features#save-snapshots-to-cloud-network-or-local-storage), [to network-attached storage or server](https://kopia.io/docs/features#save-snapshots-to-cloud-network-or-local-storage), or [locally on your machine](https://kopia.io/docs/features#save-snapshots-to-cloud-network-or-local-storage). Kopia does not 'image' your whole machine. Rather, Kopia allows you to backup/restore any and all files/directories that you deem are important or critical.
 
-* Read the [Getting Started Guide](getting-started/) for a step-by-step walkthrough of using Kopia
-* Go to the [Installation](installation/) page to get latest kopia release
-* Have a look at the [repositories](Repositories/) page to repositories usage examples
-* Check out some of Kopia's unique [Features](features/) including deduplication and encryption
-* Learn about the [Architecture](advanced/architecture/) of Kopia
-* Find more details about command line and APIs in the [Reference](reference/) section.
+Kopia has both [CLI (command-line interface)](https://kopia.io/docs/features#both-command-line-and-graphical-user-interfaces) and [GUI (graphical user interface)](https://kopia.io/docs/features#both-command-line-and-graphical-user-interfaces) versions, making it the perfect tool for both advanced and regular users. You can read more about Kopia's unique [features](features/) -- which include [compression](https://kopia.io/docs/features#compression), [deduplication](https://kopia.io/docs/features#backup-files-and-directories-using-snapshots), and [end-to-end 'zero knowledge' encryption](https://kopia.io/docs/features#end-to-end-zero-knowledge-encryption) -- and learn more about Kopia's [architecture](advanced/architecture/) to get a better understanding of how Kopia works.
```

Comment:
nit: don't use https://kopia.io in links - use only relative ones so that the Netlify preview works well.

Reply:
Updated to relative paths for this PR and the other PRs as well.